src/lesson1/HomeWorkApp.java

```diff
@@ -0,0 +1,42 @@
+package lesson1;
+
+public class HomeWorkApp {
+    public static void main(String[] args) {
+        printThreeWords();
+        checkSumSign();
+        printColor();
+        compareNumbers();
+    }
+
+    private static void printThreeWords() {
+        System.out.println("Orange");
+        System.out.println("Banana");
+        System.out.println("Apple");
+    }
+
+    private static void checkSumSign() {
+        int a = 5;
+        int b = 10;
+        String result = a+b>=0 ? "Сумма положительная" : "Сумма отрицательная";
```

Comment:
```suggestion
        String result = (a+b >= 0) ? "Сумма положительная" : "Сумма отрицательная";
```
для улучшения читаемости кода
Используйте ctrl+alt+L для автоформатирования кода